service/raft/raft_group0.cc

```diff
@@ -268,17 +268,13 @@ raft_server_for_group raft_group0::create_server_for_group0(raft::group_id gid,
 
     // initialize the corresponding timer to tick the raft server instance
     auto ticker = std::make_unique<raft_ticker_type>([srv = server.get()] { srv->tick(); });
-    lowres_clock::duration default_op_timeout = std::chrono::minutes(1);
-    if (const auto ms = utils::get_local_injector().inject_parameter<int64_t>("group0-raft-op-timeout-in-ms"); ms) {
-        default_op_timeout = std::chrono::milliseconds(*ms);
-    }
     return raft_server_for_group{
         .gid = std::move(gid),
         .server = std::move(server),
         .ticker = std::move(ticker),
         .rpc = rpc_ref,
         .persistence = persistence_ref,
-        .default_op_timeout = default_op_timeout
+        .default_op_timeout_in_ms = qp.proxy().get_db().local().get_config().group0_raft_op_timeout_in_ms
```

Comment:
My preference would be to keep the value in the `raft_server_for_group` as `lowres_clock::duration` and only make the conversion to `std::chrono::milliseconds` here (from the config value) - would also make the PR smaller as it wouldn't need to touch e.g. the "raft_group_registry.cc"

Reply:
I don't think we can do this because I wanted to make this a **live-updatable** configuration parameter, meaning that the number of milliseconds can change later.

Reply:
I meant only keeping the `raft_server_for_group.default_op_timeout` as it was (or do we need to update that value as well? I thought it will be changed just via the `cfg.group0_raft_op_timeout_in_ms`). I.e., doing this here:
```suggestion
        .default_op_timeout = std::chrono::milliseconds(qp.proxy().get_db().local().get_config().group0_raft_op_timeout_in_ms)
```
and keeping the "raft_group_registry.cc" code as it was before ... wouldn't that work the same?

Reply:
Oh, ok, that makes sense, didn't know about the `updatable_value` behavior ... that is quite handy indeed.